wigwag/system/bin/info

```diff
@@ -141,35 +196,30 @@ system() {
 	MIN15="$(echo $LOAD | awk '{ print $3}')"
 	TASKS="$(echo $LOAD | awk '{ print $4}')"
 	IPADDRESS=$(ifconfig  | grep 'inet addr:'| grep -v '127.0.0.1' | cut -d: -f2 | awk '{ print $1}')
-	test -e /lib/systemd/system/deviceos-wd.service
-	local dwd_file=$?
-	systemctl is-active --quiet deviceos-wd.service
-	local dwd_active=$?
-	systemctl is-enabled --quiet deviceos-wd.service
-	local dwd_enabled=$?
-	if [[ $dwd_active -eq 0 ]]; then
-		WDOGPID="active"
-	elif [[ $dwd_file -ne 0 ]]; then
-		WDOGPID=""
-	elif [[ $dwd_enabled -ne 0 ]]; then
-		WDOGPID="disabled"
-	else
-		WDOGPID=""
-	fi
+	readarray -t IPADDRESSES <<< $IPADDRESS
```

Comment:
Leaving a comment for future me.

`readarray` doesn't potentially hang in the same way that `read` would if stdin is empty.